src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -31,6 +32,10 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>) {
   function commitWork(finishedWork : Fiber) : void {
     switch (finishedWork.tag) {
       case ClassComponent: {
+        if (finishedWork.callbackList) {
+          callCallbacks(finishedWork.callbackList);
+          finishedWork.callbackList = null;
```

Comment:
I tend to prefer extract such lists off the object and reset it to null before calling it. Makes it easier to reason about things that happen inside of the callbacks, such as new callbacks being scheduled onto this list or errors being thrown.